modules/apps/frontend-theme/kyoung-liferay-theme/src/templates/navigation.ftl

```diff
@@ -0,0 +1,10 @@
+<#if has_navigation && is_setup_complete>
+	<button aria-controls="navigationCollapse" aria-expanded="false" aria-label="Toggle navigation" class="navbar-toggler navbar-toggler-right" data-target="#navigationCollapse" data-toggle="collapse" type="button">
```

Comment:
We should use language keys to localize the text instead of hardcoding the english in the Freemarker template.